src/grdimage.c

```diff
@@ -35,6 +35,36 @@
 #define THIS_MODULE_NEEDS	"Jg"
 #define THIS_MODULE_OPTIONS "->BJKOPRUVXYfnptxy" GMT_OPT("Sc") GMT_ADD_x_OPT
 
+static struct GMT_KEYWORD_DICTIONARY module_kw[] = { /* Local options for this module */
+	/* separator, short_option, long_option,
+	          short_directives,    long_directives,
+	          short_modifiers,     long_modifiers */
+/* ?? -A not possible because of = usage, e.g., -Aout_img=driver ? */
+	{ 0, 'C', "cpt",
+                  "",                  "",
+                  "h,i,u,U",           "hinge,zinc,tometers,frommeters" },
```

Comment:
`                  "h,i,u,U",           "hinge,zinc,tometers,frommeters" },
`

Reading the documentation on this I think perhaps inverting these are better?  E.g., use _tounit_ and _fromunit_ since they both need to take a unit as argument . So I think `--cpt:=mine.cpt+fromunit=k` might be clearer than `--cpt:=mine.cpt+tometers=k` ?

Reply:
Yes, since tounit=k would then mean change from meters to km, consistent with the meaning you had for frommeters.